src/__tests__/controller/myprofile.test.js

```diff
@@ -0,0 +1 @@
+test.only('some test', () => {});
```

Comment:
I understand that you will have a `/myprofile` route to view/edit user's self profile and a `/profiles` route to view other users' profiles. But is there a way to write the controller functions for these routes in one file instead of making 2 different files `myprofile.js` and `profile.js`.